rest/changes_api.go

```diff
@@ -225,6 +225,8 @@ func (h *handler) handleChanges() error {
 			return err
 		}
 		feed, options, filter, channelsArray, docIdsArray, _, err = h.readChangesOptionsFromJSON(body)
+		base.LogTo("Changes+", "Changes POST request.  URL: %v, feed: %v, options: %+v, filter: %v, channels: %v, docIds: %v",
+			h.rq.URL, feed, options, filter, channelsArray, docIdsArray)
```

Comment:
I feel like this should include user info (e.g. "to") so that it can be matched up with the lower level changes processing.